components/CategoryTag/CategoryTag.styled.js

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+export const Button = styled.button`
```

Comment:
I don't think this needs to be a button (yet). 
As far as I can see there is no user interaction planned for the tag at this point. So its not necessary to use a clickable element, a span or a div would work the same and don't specify the utility:)
```suggestion
export const Button = styled.span`
```

Reply:
any thoughts?

Reply:
I changed it to span. Thanks.